hw/ip/otbn/data/bignum-insns.yml

```
@@ -734,7 +734,7 @@
       doc: The WSR to read
   doc: |
     Reads a WSR to a WDR.
-    If `wsr` isn't the index of a valid WSR, this halts with an error (TODO: Specify error code).
+    If `wsr` isn't the index of a valid WSR, this results in an error (setting bit `illegal_insn` in `ERR_BITS`)
```

nit: 
```suggestion
    If `wsr` isn't the index of a valid WSR, this results in an error (setting bit `illegal_insn` in `ERR_BITS`).
```

(and below)